### DISCUSSION TITLE
Diffrent name of metrics depends of hosting model

### DISCUSSION THREAD

**Discussion by meron1122** [open-telemetry/opentelemetry-dotnet#5866] [Q&A]
For example, http intrumentation.
When application is ASP .NET Core App metrics calls `http_client_connection_duration_seconds_bucket`. but if I add metrics in console app, thsi same metrics had a diffrent name `system_net_http_http_client_connection_duration_bucket` this is very annoying when we would like have metrics grouped by one piece -  how to unification this name?

**Comment by cijothomas** [comment]
Please share a minimal repro. It is unclear who is producing the metric.
(Also, this is best asked in the contrib repo, as this repo has no component that automatically produces metrics)

**Reply by meron1122** [reply]
I used in both app this extension to Add metrics


```csharp
    public static IHostApplicationBuilder AddMetrics(this IHostApplicationBuilder builder,string? applicationName = null)
    {
        builder.Services.AddOpenTelemetry().WithMetrics(c =>
        {
            if (applicationName != null)
            {
                c.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(applicationName));
            }

            c.AddAspNetCoreInstrumentation();
            c.AddHttpClientInstrumentation();
            c.AddRuntimeInstrumentation();
            c.AddProcessInstrumentation();
            c.AddPrometheusExporter(o => o.DisableTotalNameSuffixForCounters = true);
            c.AddMeter("Microsoft.AspNetCore.Hosting",
                "Microsoft.AspNetCore.Server.Kestrel");
        });
        return builder;
    }
 ```

**Reply by cijothomas** [reply]
Please share a minimal, runnable app (you can take the example asp.net core and console apps from getting started docs in this repo and modify it), And share the output you see when using ConsoleExporter.
